### PR TITLE
Feature/architecture docs

### PR DESCRIPTION
**Pull request by coppetti**
Added the initial project description with our architecture.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,3 +12,41 @@ The Bodacious Advisor for Cuisine Over N-joyment offers a ranked list for restau
...
+
+## Project Description
+The Bodacious Advisor for Cuisine Over N-joyment ranks the restaurants within London based on Twitter reviews.
+
+## The Bacon Approach
+Our first step is to use a dataset provided by Yelp to get reviews and their associated Star Evaluation with a parser to get this key-value pair. This new dataset is used to train algorithms that are used for sentiment analysis so that we can read tweets for restaurants and predict its Star Evaluation (*-tweet). Reviews from Yelp will only be used for training the Ranking Algorithm. 
+
+Using a list with 2380 restaurants based in London with Tweeter account, we use Twitter-API to get tweets that mention those accounts so that we can apply our sentiment analysis algorithm on them, discovering their *-tweet. Our time window is three months from now, so we will not consider old tweets.
+
+//toDo ranking algorithm (Sara)
```
Comment (m-maga) [inline README.md]: toDo machine learning

**Review by m-maga — APPROVED** [review]
See my comment

**Review by h314to — APPROVED** [review]
Good! Merge away!

**Review by alesr — APPROVED** [review]

**Comment by alesr** [comment]
LGTM. Well done!

File: README.md (hunk 1 above)
Comment (coppetti) [reply README.md]: Done

**Pull request merged**